Updating Minikube and Kubectl versions.

brandoncole opened the pull request:
- Kubectl version updated to 1.4.0
- Minikube updated to 0.11.0 which is compatible with 1.4.0
- Restored the SHA256 checks which appear to have gotten removed at some point.

Commit:
Updating Minikube and Kubectl versions.

- Kubectl version updated to 1.4.0
- Minikube updated to 0.11.0 which is compatible with 1.4.0
- Restored the SHA256 checks which appear to have gotten removed at some point.

chrislovecnm reviewed (changes requested):
So kube is at 1.4.4?? Please update appropriately.

brandoncole commented:
I was trying to keep the version of kubectl in sync with the minikube version to avoid any compatibility issues.  If we want to we can use Minikube 0.12.0 with Kubectl 1.4.3.   Using 1.4.4 version of Kubectl with an older Minikube K8s version shouldn't hurt, but I considered it a nice requirement to have them in sync.  Let me know if you want them out of sync with 1.4.4 or in sync with 0.12.0 + 1.4.3

Commit:
Changing Kubectl and Minikube versions

- Using 1.4.4 for Kubectl
- Using 0.12.0 for Minikube (K8s version 1.4.3)
- Although they're out of sync, since Kubectl >= Minikube K8s should be OK

chrislovecnm commented:
LGTM
